repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -186,27 +203,38 @@ class Interpreter(prompt0: Ref[String],
     }
   }
 
-  def processLine(code: String, stmts: Seq[String], fileName: String): Res[Evaluated] = for{
-    _ <- Catching { case ex =>
-      Res.Exception(ex, "Something unexpected went wrong =(")
-    }
+  def processLine(code: String,
+                  stmts: Seq[String],
+                  fileName: String,
+                  preprocess: Preprocessor = Preprocessor(compiler.parse)
```

Comment:
In fact, what's the purpose of moving it all the way in here to begin with? Could we leave it in the old position and just make it's initialization `lazy`?